gotestskip.go

```diff
@@ -100,10 +111,11 @@ func mainErr() error {
 		}
 		return nil
 	}
-	skip, err := readSkipConfig(skipFile)
+	skip, err := readSkipConfig(skipFile, os.Getenv("GO_SKIP_ROOT"))
```

Comment:
Shouldn't this also be a flag, not just a variable?

The pattern seems to be to add a variable and default to value in environment variable.

Reply:
I agree in principle, but the problem with that is that all the args (including flags) are being passed to `go test`, so extracting flags is gonna be awkward. Hence the env var approach.